packages/tools/devtools/devtools-view/src/test/SettingsView.test.tsx

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// Normal usage pattern for @testing-library/jest-dom
+// eslint-disable-next-line import/no-unassigned-import
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import { userEvent } from "@testing-library/user-event";
+import { debug } from "jest-preview";
+import React from "react";
+
+import { SettingsView } from "../components/index.js";
+
+import { assertNoAccessibilityViolations } from "./utils/index.js";
+
+describe("SettingsView Accessibility Check", () => {
+	it("SettingsView is accessible", async () => {
+		const { container } = render(<SettingsView />);
+		await assertNoAccessibilityViolations(container);
+	});
+
+	it("Can tab/arrow navigate through the SettingsView", async () => {
+		render(<SettingsView />);
+
+		const user = userEvent.setup();
+		// Focus on the first interactive element (dropdown theme selector)
+		await user.tab();
+		const dropdown = screen.getByRole("combobox", { name: /theme selection dropdown/i });
+		expect(dropdown).toHaveFocus();
+		// The dropdown theme options are divs but get compiled as buttons in the expect function and thus fail any expect calls.
+		await user.keyboard("{Enter}");
+		await user.tab();
+		const privacyLink = screen.getByRole("link", { name: /microsoft privacy statement/i });
+		expect(privacyLink).toHaveFocus();
```

Comment:
Nit: I'm curious if there's a way to get the control that has focus, and then validate that it is the expected one. I understand this approach works, it just feels a bit backwards.

Reply:
@alexvy86  Yeah I believe we'd be writing something like `expect(document.activeElement).toBe(privacyLink);`

I feel `.toHaveFocus()` reads better, what do you think?